webcompat/templates/layout.html

```diff
@@ -36,7 +36,14 @@
 <script src="{{ url_for('static', filename='js/lib/autogrow-textfield.js') }}"></script>
 <script src="{{ url_for('static', filename='js/lib/bugform-validation.js') }}"></script>
 <script src="{{ url_for('static', filename='js/lib/bugform.js') }}"></script>
+
+{%- if ab_active('exp') == 'form-v2' %}
+<script src="{{ url_for('static', filename='js/lib/popup.js') }}"></script>
```

Comment:
Please use the same name convention we decided to keep with the rest of the filenames eg:
`issue-wizard-popup.js`
`issue-wizard-slider.js`

Also is there any reason not to combine them to one file just to avoid making things even more complicated with the 2 different codebases?